README.md

```diff
@@ -34,14 +36,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+The difference is, that a responsive website will automatically adapt to a users screen size and device and has kind of a liquid design
+
 2. Describe what it means to be mobile first vs desktop first.
 
+Desktop first means that the CSS maincode is written for devices like desktop pc's and Laptops and you build mediaqueries for the smaller devices. mobile first
+is the exact opposite 
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+1 rem normaly means 16px. So when i put `font-size: 62.5%` into the tag it means eveything inside of 'html' has now a font-size of 16px * 0,625
```

Comment:
62.5% of the root element font size. which is 16px by default (unless explicitly stated).